File: pinyougouparent/pinyougou-seckill-web/src/main/webapp/js/service/seckillGoodsService.js
app.service("seckillGoodsService", function ($http){

    this.list = function ()
    {
        return $http.get("/seckill/goods/list.do");
    }

    this.findOne = function (id)
    {
        return $http.get("/seckill/goods/findOne.do?id=" + id);
    }

    this.add = function (id)
    {
        return $http.get("/seckill/order/add.do?id=" + id);
    }
});

File: pinyougouparent/pinyougou-seckill-interface/src/main/java/com/pinyougou/seckill/service/SeckillGoodsService.java
package com.pinyougou.seckill.service;

import com.pinyougou.pojo.TbSeckillGoods;

import java.util.List;

public interface SeckillGoodsService {

    /**
     * 秒杀商品列表查询
     * @return
     */
    List<TbSeckillGoods> list();

    /**
     * 根据id查询秒杀商品
     * @param id
     * @return
     */
    TbSeckillGoods findOne(Long id);
}
